uiowa/uiowa
# Feature: work to integrate card updates from UIDS

This pr contains work to integrate the work from https://github.com/uiowa/uids/pull/533 into `uids_base`. 

Resolves https://github.com/uiowa/uiowa/issues/3416. 

# How to test

```
rm -rf ./node_modules
yarn cache clean
yarn install
yarn workspace uids_base gulp --development
yarn workspace admissions_core build
```

1. Test on all 6 of the card-based pages linked on https://sandbox.local.drupal.uiowa.edu/card-and-event-background-testing, view on desktop and mobile and look for any visual regressions. 
2. Test on a `v2` site like https://hawkeyemarchingband.local.drupal.uiowa.edu/people-0.  Try adding articles, events, and featured content to that page and test mobile and desktop views. 
3. Test on https://admissions.local.drupal.uiowa.edu/academics/actuarial-science and verify that the "Admission Process" card images remain small on mobile. 
4. Test mobile and desktop versions of https://admissions.local.drupal.uiowa.edu/counselors and verify that circle image remains the same size on desktop and mobile. 
5. Test mobile and desktop versions of https://admissions.local.drupal.uiowa.edu/student-profiles and verify that circle image remains the same size with spacing inside the enclosed card. 
6. Test mobile and desktop views of https://grad.local.drupal.uiowa.edu/academics/thesis-defenses/list, https://grad.local.drupal.uiowa.edu/dei/srop/past-scholars/2019, https://grad.local.drupal.uiowa.edu/dei/srop/mentors
